Assets/_BForBoss/_Core/Scripts/Boss/DerekBossManager.cs

```diff
@@ -31,55 +57,102 @@ public void UpdatePhase(DerekContextManager.Phase phase)
-                    //Find Closest Death Wall and activate - 
-                    //Start Movement with new given rotation (i.e. clockwise/anticlockwise)
-                    //Give small buffer of wait time - then StartMissile behavior
-                    //Ensure Derek is invincible in this case (edge case)
-                    
-                    Debug.Log("Raising Shields");
-                    Debug.Log("Resetting Position");
-                    Debug.Log("Activating Death Wall Closest to player");
-                    Debug.Log("Rotating to make sure death wall gets closer to player");
-                    Debug.Log("Starting Homing Missiles");
-                    Debug.Log("------------------------");
-                    
+                    _isVulnerable = false;
+                    _shieldBehaviour.ToggleShield(true);
+                    foreach (DerekMissileLauncherBehaviour missileLauncher in _missileLauncherBehaviours)
+                    {
+                        missileLauncher.StartShooting(_timeBetweenMissileShots);
+                    }
+                    _wipeoutWallsManager.ActivateClosestLongWallAndRotate();
                     break;
+                //End Missile Behavior
+                //Deactivate all DeathWalls
+                //Lower Shields
+                //Ensure Derek is susceptible to damage
                 case DerekContextManager.Vulnerability.Vulnerable:
-                    //End Missile Behavior
-                    //Deactivate all DeathWalls
-                    //Lower Shields
-                    //Ensure Derek is susceptible to damage
-                    
-                    Debug.Log("Stopping Missiles");
-                    Debug.Log("Deactivating Death Walls");
-                    Debug.Log("Lowering Shields");
-                    Debug.Log("------------------------");
+                    _isVulnerable = true;
+                    _vulnerabilityTimer = _vulnerabilityDuration;
+                    foreach (DerekMissileLauncherBehaviour missileLauncher in _missileLauncherBehaviours)
+                    {
+                        missileLauncher.StopShooting();
+                    }
+                    _wipeoutWallsManager.DeactivateWallAndRotation();
+                    _shieldBehaviour.ToggleShield(false);
                     break;
             }
         }
+        
+        //Temporary Damage Receiving Functionality - Will be removed in BFB-516 with Derek Health Behaviour component
+        public void OnCollided(Vector3 collisionPoint, Vector3 collisionNormal)
+        {
+            if (_shieldBehaviour.IsActive())
+            {
+                Perigon.Utility.Logger.LogWarning("Boss was shot even with shields up, which shouldn't be possible, ignoring damage", LoggerColor.Yellow, "derekboss");
+                return;
+            }
+            
+            _onVulnerabilityExpired?.Invoke(true);
+        }
+
+        private void OnValidate()
```

Comment:
I personally think we should use OnAwake for ingame configuration since this function is only called within the unity editor and not the executable game. 
https://github.com/Demigiant/dotween/issues/539

So if there's an issue, and we build it, the build itself won't call this.

Reply:
Good point